test/e2e/autoscaling/cluster_size_autoscaling.go

```diff
@@ -363,7 +364,7 @@ var _ = SIGDescribe("Cluster size autoscaling [Slow]", func() {
 		if status.target != target {
 			klog.Warningf("Final number of nodes (%v) does not match initial scale-up target (%v).", status.target, target)
 		}
-		framework.ExpectEqual(status.timestamp.Add(freshStatusLimit).Before(time.Now()), false)
+		gomega.Expect(status.timestamp.Add(freshStatusLimit)).ToNot(gomega.BeTemporally("<", time.Now()))
```

Comment:
I think this becomes simpler when avoiding the `ToNo`:

```suggestion
		gomega.Expect(status.timestamp.Add(freshStatusLimit)).To(gomega.BeTemporally(">=", time.Now()))
```